pkg/proxy/iptables/proxier.go

```diff
@@ -1572,7 +1572,17 @@ func (proxier *Proxier) syncProxyRules() {
 	err = proxier.iptables.RestoreAll(proxier.iptablesData.Bytes(), utiliptables.NoFlushTables, utiliptables.RestoreCounters)
 	if err != nil {
 		glog.Errorf("Failed to execute iptables-restore: %v", err)
-		glog.V(2).Infof("Rules:\n%s", proxier.iptablesData.Bytes())
```

Comment:
Actually, we already log them at V(5) a few lines earlier whether there's an error or not. So we could just drop the logging here.